src/rgw/rgw_op.h

```diff
@@ -1773,9 +1773,8 @@ class RGWGetObjLayout : public RGWOp {
   rgw_raw_obj head_obj;
 
 public:
-  RGWGetObjLayout() {
-    delete target;
-  }
+  RGWGetObjLayout() = default;
+  ~RGWGetObjLayout() = default;
```

Comment:
That's right. I have already gave those comments earlier, but I'm not able to see them in this PR now.